examples_tests/13.BRDF_Explorer/CMakeLists.txt

```diff
@@ -4,8 +4,14 @@ if(NOT RES)
 	message(FATAL_ERROR "common.cmake not found. Should be in {repo_root}/cmake directory")
 endif()
 
+set(BRDF_SOURCES
+	"../../ext/CEGUI/CEGUI.cpp"
```

Comment:
because of the pure insane number of dependencies, `irr::ext::cegui` should be its own statically linked library

Reply:
Will compile ext as lib and link later, when things render on the screen.

Reply:
Added to task list in #279